data_steward/common.py

```diff
@@ -196,3 +196,13 @@
 MAPPING_PREFIX = '_mapping_'
 EXT = 'ext'
 EXT_SUFFIX = '_ext'
+
+ACTIVITY_SUMMARY = 'activity_summary'
+HEART_RATE_MINUTE_LEVEL = 'heart_rate_minute_level'
+HEART_RATE_SUMMARY = 'heart_rate_summary'
+STEPS_INTRADAY = 'steps_intraday'
+
+FITBIT_TABLES = [
+    ACTIVITY_SUMMARY, HEART_RATE_MINUTE_LEVEL, HEART_RATE_SUMMARY,
+    STEPS_INTRADAY
+]
```

Comment:
I added this in DC-1001, will this cause any issues during the merge? If so, should we keep yours or mine?

Reply:
I added these for the integration tests to work. You can merge yours first and then I will rebase